File: benchmark/workloads/csharp/objectArray.cs
var values = new Box[200_000];
for (var index = 0; index < values.Length; index++)
{
    values[index] = new Box(index + 1);
}
long total = 0;
foreach (var value in values)
{
    total += value.Value;
}
Console.WriteLine(total);

sealed class Box
{
    public Box(long value) => Value = value;
    public long Value { get; }
}

File: benchmark/workloads/ruby/objectArray.rb
class Box
  attr_reader :value

  def initialize(value)
    @value = value
  end
end

values = Array.new(200_000) { |index| Box.new(index + 1) }
total = 0
index = 0
while index < values.length
  total += values[index].value
  index += 1
end
puts total
